pages/introduction.vue

```diff
@@ -0,0 +1,131 @@
+<template>
+  <div id="main-content" class="text-wrapper w-full">
+    <h1 class="text-center mb-6">Introduction to Scottish Witchcraft</h1>
+
+    <p class="mb-3 text-justify">
+      This interactive map displays the known residences and detention, trial,
+      and death locations of over 3000 accused witches in Scotland, based upon
+      information provided by
+      <a
+        href="https://witches.hca.ed.ac.uk/"
+        class="underline hover:text-gray-500"
+      >
+        The Survey of Scottish Witchcraft </a
+      >.
+    </p>
+    <div class="flex justify-center mb-3">
+      <iframe
+        id="kaltura_player"
+        type="text/javascript"
+        src='https://cdnapisec.kaltura.com/p/2010292/embedPlaykitJs/uiconf_id/55171522?iframeembed=true&entry_id=1_yj5c097l&config[provider]={"widgetId":"1_buq8663s"}'
+        style="width: 400px; height: 285px; border: 0"
```

Comment:
can we make this video bigger? Ideally it would be good to increase the spacing at the top and bottom of the video and have the video the same width as the text body. Thanks Ruby

Reply:
done, does look better thanks

Reply:
Brilliant. Thanks Ruby. Marking as resolved